content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -50,6 +50,10 @@ See [Studio](./command-reference#studio) documentation for more information.
 
 ## Prisma CLI
 
+### <inlinecode>PRISMA_HIDE_PREVIEW_FLAG_WARNINGS</inlinecode>
+
+`PRISMA_HIDE_PREVIEW_FLAG_WARNINGS` is used to hide the warning message stating that a preview feature flag can be removed. It is a truthy value.
```

Comment:
```suggestion
`PRISMA_HIDE_PREVIEW_FLAG_WARNINGS` hides the warning message that states that a preview feature flag can be removed. It is a truthy value.
```

Reply:
My suggestion above is to bring this in line with our style guide (avoiding passive voice). I realize that other instances on the page use the form "is used to...", but moving forwards, we want new content to comply - we will come back to existing content at a later date.